src/screens/home/views/OutbreakExposedView.tsx

```diff
@@ -19,7 +19,10 @@ export const OutbreakExposedView = () => {
   const historyItem: OutbreakHistoryItem = currentOutbreakHistory[0];
 
   const severity = historyItem?.severity;
-  const exposureDate = formatExposedDate(new Date(historyItem?.checkInTimestamp), dateLocale);
+  const exposureDate =
+    historyItem?.checkInTimestamp === undefined
+      ? formatExposedDate(new Date(), dateLocale)
```

Comment:
We should only force a date if we're in TEST_MODE 

@smcmurtry fallback scenario - don't show a date ?  What do you think?

Reply:
updated